### ISSUE TITLE
The 6.0.0 early releases are not following Gem-compatible versioning scheme?

### ISSUE THREAD

**Issue by jeremycole** [rails/webpacker#2940]
I guess this might be known already (surely someone will have seen it in their own project by now) but it seems that the versioning scheme from 6.0.0 before final release is not compatible with Gem and is thus confusing Dependabot at least.

Due to this, Dependabot is suggesting an _upgrade_ from `6.0.0.beta.5` to `6.0.0.pre.2` (which is older):

```
> ['6.0.0.pre.2', '6.0.0.beta.5'].map { |v| Gem::Version.new(v) }.sort.map(&:to_s)
=> ["6.0.0.beta.5", "6.0.0.pre.2"]
```

I'm not sure if there's a simple solution to this until a real 6.0.0 release is made (which seems that it will supercede `6.0.0.pre.2` but I guess in general dependabot (and perhaps bundler itself?) will ignore `beta` releases and continue suggesting this downgrade to the last `pre` release.

**Comment by coder2000** [comment]
Specifying `~> 6.0.0.beta.6` in your Gemfile still installs `6.0.0.pre.2` which is older and can cause problems when you expect the newest version.

**Comment by donv** [comment]
Maybe yank versions "6.0.0.pre.1" and "6.0.0.pre.2" ?

**Comment by dhh** [comment]
Fixed with 6.0.0.rc1
